test/images/issue-74839/README.md

```diff
@@ -0,0 +1,7 @@
+# Reproduction of k8s issue #74839
+
```

Comment:
Would be great to have a one-line description here, even better if we note down what the test does from a high level :)